first_order_model/frames_dataset.py

```diff
@@ -7,7 +7,7 @@
 import numpy as np
 from torch.utils.data import Dataset
 import pandas as pd
-from augmentation import AllAugmentationTransform
+#from augmentation import AllAugmentationTransform
```

Comment:
issue with paths, please fix this rather than comment it out. It is a relative import, replace with absolute